Microsoft.Toolkit.Uwp.SampleApp/SamplePages/SystemInformation/SystemInformationPage.xaml

```diff
@@ -206,6 +206,30 @@
                 <StackPanel x:Name="Stack16"
                             Margin="10"
                             Orientation="Horizontal">
+                    <TextBlock FontSize="18"
+                               FontWeight="Bold"
+                               Text="Last reset time:"
+                               TextWrapping="Wrap" />
+                    <TextBlock Margin="20,0"
+                               FontSize="18"
+                               Text="{x:Bind LastResetTimeProp, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}"
+                               TextWrapping="Wrap" />
+                </StackPanel>
+                <StackPanel x:Name="Stack17"
+                            Margin="10"
+                            Orientation="Horizontal">
+                    <TextBlock FontSize="18"
+                               FontWeight="Bold"
+                               Text="Current launch count:"
+                               TextWrapping="Wrap" />
+                    <TextBlock Margin="20,0"
+                               FontSize="18"
+                               Text="{x:Bind LaunchCountProp, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}"
+                               TextWrapping="Wrap" />
+                </StackPanel>
+                <StackPanel x:Name="Stack18"
```

Comment:
Do we actually need to name all these stack panels?

Reply:
I don't see why there is a name. I followed the previous pattern.